evm/deploy-amplifier-gateway.js

```diff
@@ -260,10 +260,10 @@ async function deploy(config, chain, options) {
                 error = true;
             }
 
-            if (signerHashByEpoch !== signerHash) {
+            if (signersHashByEpoch !== signersHash) {
                 printError(`ERROR: Signer hash mismatch for signer set ${i + 1}`);
```

Comment:
you can release a snapshot to test that this will work correctly before a proper release

Reply:
done